stream/distributedlog/core/src/main/java/org/apache/distributedlog/LocalDLMEmulator.java

```diff
@@ -308,13 +308,15 @@ public static URI createDLMURI(String zkServers, String path) throws Exception {
     public static void main(String[] args) throws Exception {
         try {
             if (args.length < 1) {
-                System.out.println("Usage: LocalDLEmulator <zk_port>");
+                System.out.println("Usage: LocalDLEmulator <zk_host> <zk_port>");
                 System.exit(-1);
             }
 
-            final int zkPort = Integer.parseInt(args[0]);
+            final String zkHost = args[0];
+            final int zkPort = Integer.parseInt(args[1]);
```

Comment:
Can we make it compatible with only version ?
Just handle the case in which you have only one parameter

Reply:
Done 😄